docs/authentication/jwt-auth.html

```diff
@@ -16,13 +16,407 @@
+// Users credentials are authenticated</code></pre>
+    <p>At this stage, you would normally hash the password and verify it against a database.
+        However, for simplicity, we are going to assume the user successfully logged in.</p>
+    <div class="underline"></div>
+    <h2 class="heading-2"><span class="blue-text">Step 5:</span> Create the signed JWT</h2>
+    <p class="block-text">A JWT contains claims about the user that we want include in subsequent request.
+         You can specify any information as a claim, however there are "Registered Claims" which have a pre-defined meaning:</p>
+         <ul>
+           <li>iss: The issuer of the token </li>
+           <li>sub: The subject of the token</li>
+           <li>aud: The audience of the token</li>
+           <li>exp: The expiration time which MUST be after the current date/time.</li>
+           <li>nbf: Defines the time before which the JWT MUST NOT be accepted for processing</li>
+           <li>iat: The time the JWT was issued. Can be used to determine the age of the JWT</li>
+           <li>jti: Unique identifier for the JWT. Can be used to prevent the JWT from being replayed.</li>
+         </ul>
+    <p>SwiftJWT comes with a struct representing these Registered Claims which we will use for our example.</p>
+    <p>Beneath the users credentials, initialize the users claims:</p>
+    <pre><code class="language-swift">let myClaims = ClaimsStandardJWT(iss: "Kitura", sub: credentials.username, exp: Date(timeIntervalSinceNow: 3600))</code></pre>
+    <p>These claims tell us users username, that they were authenticated by Kitura and we have set the token to expire in one hour.</p>
+    <p>Beneath our claims we initialize our JWT:</p>
+    <pre><code class="language-swift">var myJWT = JWT(claims: myClaims)</code></pre>
+    <p>We can sign this JWT using the JWTSigner we created in step 2:</p>
+    <pre><code class="language-swift">let signedJWT = try myJWT.sign(using: App.jwtSigner)</code></pre>
+    <p>Finally we return the signed JWT String to the User:</p>
+    <pre><code class="language-swift">response.send(signedJWT)</code></pre>
+    <p>Our completed login route should look as follows:</p>
+    <pre><code class="language-swift">app.router.post("/jwtlogin") { request, response, next in
+    let credentials = try request.read(as: UserCredentials.self)
+    // Users credentials are authenticated
+    let myClaims = ClaimsStandardJWT(iss: "Kitura", sub: credentials.username, exp: Date(timeIntervalSinceNow: 3600))
+    var myJWT = JWT(claims: myClaims)
+    let signedJWT = try myJWT.sign(using: App.signer)
+    response.send(signedJWT)
+    next()
+}</code></pre>
+<div class="underline"></div>
+
+<h2 class="heading-2"><span class="blue-text">Step 6:</span> Test the JWT creation</h2>
+<p>Compile your project and start the server.</p>
+<p>To test the route using curl, open Terminal and enter the following:</p>
+<pre><code>curl -X POST \
```

Comment:
Please can you indent the curl command as we did in the routing guides?

Reply:
Have indented lines below comment.